addons/mass_mailing/models/mass_mailing.py

```diff
@@ -819,3 +820,12 @@ def _process_mass_mailing_queue(self):
                 mass_mailing.send_mail()
             else:
                 mass_mailing.state = 'done'
+
+    def append_style(self, body):
+        files, remains = self.env['ir.qweb']._get_asset_content('mass_mailing.assets_mail_themes_common', {})
+        assets = AssetsBundle('mass_mailing.assets_mail_themes_common', files, remains, env=self.env)
+
+        scss_css_styles = assets.preprocess_css()
+        style_in_head = "<head><style>" + scss_css_styles + "</style></head>"
```

Comment:
It'd be great to compose this in a separate method to be able to insert compatibility snippets (for outlook sake mainly) in the header like:
```html
<!--[if gte mso 15]>
        <xml>
            <o:OfficeDocumentSettings>
            <o:AllowPNG/>
            <o:PixelsPerInch>96</o:PixelsPerInch>
            </o:OfficeDocumentSettings>
        </xml>
        <![endif]-->
```